docs/config/shared-options.md

```diff
@@ -145,7 +145,7 @@ A package with conditional exports may have the following `exports` field in its
 
 Here, `import` and `require` are "conditions". Conditions can be nested and should be specified from most specific to least specific.
 
-Vite has a list of "allowed conditions" and will match the first condition that is in the allowed list. The default allowed conditions are: `import`, `module`, `browser`, `default`, and `production/development` based on current mode. The `resolve.conditions` config option allows specifying additional allowed conditions.
+Vite has a list of "allowed conditions" and will match the first condition that is in the allowed list. The default allowed conditions are: `import`, [`module`](https://webpack.js.org/guides/package-exports/#providing-commonjs-and-esm-version-stateless), `browser`, `default`, and `production/development` based on current mode. The `resolve.conditions` config option allows specifying additional allowed conditions.
```

Comment:
It looks like Vite doesn't follow the logic from the link. Webpack uses `module` for `require()` to prevent code duplication, but Vite would skip `module` for `require()`, and only works for `import`. Maybe we should remove the link for now.

Reply:
Hmm. I wonder if `module` is something we should support in that case then. Anyway, I'll go ahead and close this based on your feedback